packages/pages/src/components/breadcrumbs/types.ts

```diff
@@ -0,0 +1,8 @@
+import { ReactNode } from "react";
+
+export interface BreadcrumbsProps {
+  separator?: ReactNode;
+  className?: string;
+  yaTrack?: string;
```

Comment:
we don't need ya track/scope McGin made an analytics component that handles sending those events differently